### PR TITLE
Grid

### PR DESCRIPTION
**Pull request by parulgupta2001**

### CODE REVIEW

File: docs/grid.html [hunk 1]
```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (junaidshaikh-js) [inline docs/grid.html:7]: The title of the page should be meaningful

File: docs/grid.html [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+            <li><a href="rating.html">Rating</a></li>
+            <li><a href="list.html">List</a></li>
+            <li><a href="modal.html">Modal</a></li>
+            <li><a href="toast.html">Toast</a></li>
+            <li><a href="grid.html">Simplified Grid</a></li>
+
+        </ul>   
+    </aside>
+
+    <div class="main ">
```
Comment (junaidshaikh-js) [inline docs/grid.html:42]: Main content of the page should be nested inside the main element.

**Review by junaidshaikh-js — COMMENTED** [review]
Good Work! 

Suggested changes can be implemented. 
Everything else is fine.

**Review by junaidshaikh-js — APPROVED** [review]

**Comment by jDineshKarthic** [comment]
Looks great just keep an eye on the line spacing.

**Review by jDineshKarthic — APPROVED** [review]

**Review by jDineshKarthic — APPROVED** [review]

**Review by jDineshKarthic — APPROVED** [review]

**Comment by jDineshKarthic** [comment]
writing better names for classes and ids will help others understand better

**Pull request merged**